examples/DISCO_IOT_BeaconDemo/DISCO_IOT_BeaconDemo.ino

```diff
@@ -0,0 +1,86 @@
+/*
+
+ DISCO_IOT_BeaconDemo
```

Comment:
This example is not really linked to the board so perhaps we can have a more generic name ?

Reply:
It is linked to the board because of the pin configuration:

SPIClass SPI_3(44, 43, 42);
SPBTLERFClass BTLE(&SPI_3, 50, 57, 31, LED4);